modules/hcp-dr-oadp-backup-cp-workload-bm.adoc

```diff
@@ -0,0 +1,164 @@
++
+[source,terminal]
+----
+$ oc --kubeconfig <management_cluster_kubeconfig_file> \
+  patch nodepool -n <hosted_cluster_namespace> <node_pool_name> \
+  --type json -p '[{"op": "add", "path": "/spec/pausedUntil", "value": "true"}]'
+----
+
+. Pause the reconciliation of the `AgentCluster` resource by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <management_cluster_kubeconfig_file> \
+  annotate agentcluster -n <hosted_control_plane_namespace>  \
+  cluster.x-k8s.io/paused=true --all
+----
+
+. Pause the reconciliation of the `AgentMachine` resource by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <management_cluster_kubeconfig_file> \
+  annotate agentmachine -n <hosted_control_plane_namespace>  \
+  cluster.x-k8s.io/paused=true --all
+----
+
+. If you are backing up and restoring to the same management cluster, annotate the `HostedCluster` resource to prevent the deletion of the hosted control plane namespace by running the following command:
++
+[source,terminal]
+----
+$ oc --kubeconfig <management_cluster_kubeconfig_file> \
+  annotate hostedcluster -n <hosted_cluster_namespace> <hosted_cluster_name> \
+  hypershift.openshift.io/skip-delete-hosted-controlplane-namespace=true
+----
+
+. Create a YAML file that defines the `Backup` CR:
++
+.Example `backup-control-plane.yaml` file
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: velero.io/v1
+kind: Backup
+metadata:
+  name: <backup_resource_name> # <1>
+  namespace: openshift-adp
+  labels:
+    velero.io/storage-location: default
+spec:
+  hooks: {}
+  includedNamespaces: # <2>
+  - <hosted_cluster_namespace> # <3>
+  - <hosted_control_plane_namespace> # <4>
+  - <agent_namespace> # <5>
+  includedResources:
+  - sa
+  - role
+  - rolebinding
+  - pod
+  - pvc
+  - pv
+  - bmh
+  - configmap
+  - infraenv
+  - priorityclasses
+  - pdb
+  - agents
+  - hostedcluster
+  - nodepool
+  - secrets
+  - services
+  - deployments
+  - hostedcontrolplane
+  - cluster
+  - agentcluster
+  - agentmachinetemplate
+  - agentmachine
+  - machinedeployment
+  - machineset
+  - machine
+  excludedResources: []
+  storageLocation: default
+  ttl: 2h0m0s
+  snapshotMoveData: true # <6>
+  datamover: "velero" # <6>
+  defaultVolumesToFsBackup: true # <7>
+----
+====
+<1> Replace `backup_resource_name` with the name of your `Backup` resource.
+<2> Selects specific namespaces to back up objects from them. You must include your hosted cluster namespace and the hosted control plane namespace.
+<3> Replace `<hosted_cluster_namespace>` with the name of the hosted cluster namespace, for example, `clusters`.
+<4> Replace `<hosted_control_plane_namespace>` with the name of the hosted control plane namespace, for example, `clusters-hosted`.
+<5> Replace `<agent_namespace>` with the namespace where your Agent, BMH, and `InfraEnv` CRs are located, for example, `agents`.
```

Comment:
Maybe for consistency, either use the tilde for all of the resources or none of them

```suggestion
<5> Replace `<agent_namespace>` with the namespace where your `Agent`, `BMH`, and `InfraEnv` CRs are located, for example, `agents`.
```